source/documents.txt

```diff
@@ -13,10 +13,12 @@ Documents are individual records in a MongoDB :ref:`collection
 
 From the :guilabel:`Documents` tab, you can
 :ref:`view <compass-view-documents>`, :ref:`insert <insert-documents>`,
-:ref:`modify <compass-modify-documents>`,
-:ref:`clone <clone-documents>`, and
-:ref:`delete <compass-delete-documents>` documents in your selected
-collection or view.
+:ref:`modify <compass-modify-documents>`, 
+:ref:`modify multiple <compass-bulk-update>`,
```

Comment:
Hmm. The documentation refers to this as "modify multiple" throughout and we refer to it as update or bulk update. And for single ones we refer to it as "edit". I don't know if this matters.

Reply:
I think this could be a situation where we are describing the functionality vs the feature name. Similar to how we labeled the Generative AI stuff as "Query with Natural Language" to better articulate the use case. Do you have any preference here @jaoppenheim? Does it need to be labeled bulk updates or is "modify multiple" sufficient?

Reply:
I think modify is fine since that is the language that we already have in our docs today